src/com/company/gamecontent/GameMap.java

```diff
@@ -374,9 +391,9 @@ public boolean occupied(Rectangle givenRect, GameObject exceptObject) {
                     Rectangle objOnBlockRect = objOnBlock.getRect();
 
                     // DEBUG
-                    //Main.printMsg("Check 1: (" + thisObjRect.x + "," + thisObjRect.y + "," + thisObjRect.width + "," + thisObjRect.height);
-                    //Main.printMsg("Check 2: (" + objOnBlockRect.getX() + "," + objOnBlockRect.getY() + "," + objOnBlockRect.getWidth() + "," + objOnBlockRect.getHeight());
-                    //Main.printMsg("Check 3: (" + objOnBlock.getAbsLoc()[0] + "," + objOnBlock.getAbsLoc()[1] + "," + objOnBlock.getAbsSize()[0] + "," + objOnBlock.getAbsSize()[1]);
+//                    LOG.debug("Check 1: (" + thisObjRect.x + "," + thisObjRect.y + "," + thisObjRect.width + "," + thisObjRect.height);
```

Comment:
I see you forgot to answer my comment about it.
I asked you why did you change function calls inside these 3 commented lines. I suppose that you wrongly use the updated API from your local workspace and did not revert it before doing commit.

Reply:
Now I have done a log 2.x version 2.x import everywhere.